### PR TITLE
Added the article related to Silex PHP microframework

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: source/_posts/silex.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+5.3.17+ versions:
+
+```sh
+$ composer require ircmaxell/password_compat
+```
+
+## Usage
+
+By default, Silex hashes passwords using the `sha512` algorithm. In order to 
+use `bcrypt`, override the default settings of `security.encoder.digest`
```
Comment (xabbuh) [inline source/_posts/silex.md]: of the

File: source/_posts/silex.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Use the `encodePassword()` method of the encoder related to the user:
+
+```php
+$encoder = $app['security.encoder_factory']->getEncoder($user);
+$hashedPassword = $encoder->encodePassword($plainPassword, $user->getSalt());
+```
+
+`$user` is the object that represents the user and it must implement the
+`Symfony\Component\Security\Core\User\UserInterface` interface.
```
Comment (xabbuh) [inline source/_posts/silex.md]: doubled interface here

File: source/_posts/silex.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Use the `isPasswordValid()` method of the encoder related to the user:
+
+```php
+$encoder = $app['security.encoder_factory']->getEncoder($user);
+$isValid = $encoder->isPasswordValid($user->getPassword(), $plainPassword, $user->getSalt());
+```
+
+`$user` is the object that represents the user and it must implement the
+`Symfony\Component\Security\Core\User\UserInterface` interface.
```
Comment (xabbuh) [inline source/_posts/silex.md]: same here

File: source/_posts/silex.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+sponsor: "SensioLabs"
+sponsor_url: "https://sensiolabs.com"
+sponsor_twitter: sensiolabs
+sponsor_image: "/themes/dshafik/securepasswords.info/assets/images/sensiolabs.png"
+---
+
+## Installation
+
+Silex relies on PHP hashing algorithms to encode passwords. The recommended
+algorithm is called `bcrypt` and it's available in PHP 5.5+ via the
```
Comment (xabbuh) [inline source/_posts/silex.md]: You can probably simply this: "[...] is called `bcrypt` and is available [...]"

**Comment by stof** [comment]
:+1:

**Pull request merged**